.babelrc

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
true regarding npm jacking up the formatting.  I had just moved the babelrc in 1.4 for the same reason -> as the number of aliases grow, the natural spot to maintain them all is in the same place where we load other packages..  _my_ logic: why have our aliases hidden in babelrc... when it doesn't have to go there and we don't need to maintain multiple files..